perun-wui-core/src/main/java/cz/metacentrum/perun/wui/client/utils/JsUtils.java

```diff
@@ -290,6 +290,46 @@ public static final native String encodeBase64(String decodedString)/*-{
 		return btoa(decodedString);
 	}-*/;
 
+	/**
+	 * Decode any ASCII string encoded as base64urlSafe
+	 *
+	 * @param encodedString String to decode
+	 * @return Decoded string
+	 */
+	public static final native String decodeBase64UrlSafe(String encodedString)/*-{
+		return decodeBase64(base64UrlSafeToNonSafe(encodedString));
```

Comment:
You can't reference other native methods in GWT directly. See: http://www.gwtproject.org/doc/latest/DevGuideCodingBasicsJSNI.html
or simply create standard java method as a wrapper, which will perform passing results of first call to the second and returning results.